libkpod/container_server.go

```diff
@@ -590,6 +597,46 @@ func (c *ContainerServer) ListContainers() []*oci.Container {
 	return c.state.containers.List()
 }
 
+// ListRunningContainers returns a list of all containers that are currently running
+func (c *ContainerServer) ListRunningContainers() []*oci.Container {
```

Comment:
No, I forgot about closures.  I go caught up thinking of the haskell feature where calling a function with incomplete arguments will return a new function that only accepts the remaining arguments